charms/kfp-profile-controller/src/charm.py

```diff
@@ -70,8 +72,7 @@ def _set_pod_spec(self, event):
         self.model.unit.status = MaintenanceStatus("Setting pod spec")
 
         deployment_env = {
-            "MINIO_ACCESS_KEY": os["access-key"],
-            "MINIO_SECRET_KEY": os["secret-key"],
+            "minio-secret-kfp": {"secret": {"name": "minio-secret-kfp"}},
```

Comment:
I read this as minio's secret for kfp, not kfp's secret for minio.  Maybe flip it around?  This could also get deployed more than once in the same namespace in theory, so maybe better practice would be to do `CHARM_NAME-minio-secret` or similar?

Reply:
(eg: `"{self.model.app.name}-minio-credentials"`, or similar)